prepare_for_develop.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+mkdir Zeeguu-Web/instance
+chmod uog+wrx Zeeguu-Web/instance
+
+docker stop zeeguu-web
+docker run --net=host -d --name=zeeguu-web-with-volume -v $(pwd)/Zeeguu-Web:/opt/Zeeguu-Web zeeguu-web
+
+echo "you can make changes to Zeeguu-Web locally; then execute 'docker exec zeeguu-web-with-volume python ./Zeeguu-Web/setup.py develop' to deploy"
```

Comment:
@mfurak this is cool.
can we move the `docker exec zeeguu-web-with-volume python ./Zeeguu-Web/setup.py develop` into it's own shell script? something entitled like: './redeploy_web.sh' or so? then the echo message becomes shorter.

Reply:
Also, shouldn't this script also do with zeeguu-api-core what it does with zeeguu-web?